index.html

```diff
@@ -279,7 +279,7 @@ <h3 class="card__title">Comments</h3>
 				</label>
 				<label>
 					<input name="experienceUrl" type="radio" value="croupier"
-						data-siteUrl="https://playground.marfeel.com/experiences/branded-content/branded-content-article.html" />
+						data-siteUrl="https://playground.marfeel.com/templates/branded-content-article.html" />
```

Comment:
I don't see that you have move this card in that folder, isn't it?

Reply:
Yes I have, it's down here ↓
Maybe it's clearer if you look directly in the branch: https://github.com/Marfeel/flowcards-playground/blob/MRF-83206%2FaddHeroElementToDFPFlowcard/templates/branded-content-article.html